packages/config/lib/options.ts

```diff
@@ -6,6 +6,10 @@ interface ResolvedConfigOption {
   validation: Function
   isFolder?: boolean
   isExperimental?: boolean
+  /**
+   * Can be mutated with Cypress.config() or test-specific configuration overrides
+   */
+  isWriteable?: boolean
```

Comment:
late to the game on naming, but would `canUpdateDuringTestTime` be more descriptive?

Reply:
Updated in [f644e25](https://github.com/cypress-io/cypress/pull/18896/commits/f644e25c10593160ca97e704ade1d41af39e37cf)